uberfire-workbench/uberfire-workbench-processors/src/main/java/org/uberfire/annotations/processors/ClientEditorActivityGenerator.java

```diff
@@ -0,0 +1,304 @@
+                                  "Class name: " + className);
+            messager.printMessage(Kind.NOTE,
+                                  "Identifier: " + identifier);
+            messager.printMessage(Kind.NOTE,
+                                  "getContextIdMethodName: " + getContextIdMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onStartup0ParameterMethodName: " + onStartup0ParameterMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onStartup1ParameterMethodName: " + onStartup1ParameterMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onMayCloseMethodName: " + onMayCloseMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onCloseMethodName: " + onCloseMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onShutdownMethodName: " + onShutdownMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onOpenMethodName: " + onOpenMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onLostFocusMethodName: " + onLostFocusMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "onFocusMethodName: " + onFocusMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "getDefaultPositionMethodName: " + getDefaultPositionMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "getTitleMethodName: " + getTitleMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "getTitleWidgetMethodName: " + getTitleWidgetMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "getWidgetMethodName: " + getWidgetMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "isWidgetMethodReturnTypeElement: " + isWidgetMethodReturnTypeElement);
+            messager.printMessage(Kind.NOTE,
+                                  "isWidget: " + Boolean.toString(isWidget));
+            messager.printMessage(Kind.NOTE,
+                                  "hasUberView: " + Boolean.toString(hasUberView));
+            messager.printMessage(Kind.NOTE,
+                                  "setContentMethodName: " + setContentMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "getContentMethodName: " + getContentMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "isDirtyMethodName: " + isDirtyMethodName);
+            messager.printMessage(Kind.NOTE,
+                                  "Qualifiers: " + String.join(", ",
+                                                               qualifiers));
+        }
+
+        //Validate getWidgetMethodName and isWidget
```

Comment:
Suggestion: we could validate all 6 cases, collect errors and only throw exception afterwards. Instead of fixing one problem at a time, you could fix them all at once.

Reply:
I agree, but as I said above, it is looks to be based on the EditorActivityGenerator hence I was resistant to modify this class specifically

Reply:
Imho, if we have an opportunity to improve, we should take it. Why copy old patterns when they can be improved?
After some closer code inspection I would leave this part as is and maybe improved the setup as i suggested [here](https://github.com/kiegroup/appformer/pull/724/files#r302947276)